pkg/encoding/bigint/bigint.go

```diff
@@ -84,10 +84,12 @@ func getEffectiveSize(buf []byte, isNeg bool) int {
 //   when n == 0. For zero is equal to empty slice in NEO3.
 // https://github.com/neo-project/neo-vm/blob/master/src/neo-vm/Types/Integer.cs#L16
 func IntToBytes(n *big.Int) []byte {
```

Comment:
After both methods can be named `bigint.FromBytes`/`bigint.ToBytes`. Thoughts? @roman-khimov @AnnaShaleva

Reply:
¡Me gusta!